test/Sentry.OpenTelemetry.Tests/Sentry.OpenTelemetry.Tests.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net7.0;net6.0;netcoreapp3.1;net48</TargetFrameworks>
```

Comment:
`net6.0;net462;netstandard2.0;netstandard2.1`

I believe this is how things will get tested (how TFM resolution works):

Test TFM -> Lib TFM:
net48 -> net462
netcoreapp3.1 -> netstandard2.1
net6.0 -> net6.0
net7.0 -> net6.0

So we're not testing `netstandard2.0`. 

Worth validating it with
```
#if NETSTANDARD2_0 
throw new Exception("should fail a test");
#endif
```


You can specify which TFM you want to target like this: `    <ProjectReference Update="../../src/Sentry/Sentry.csproj" SetTargetFramework="TargetFramework=netstandard2.0" />
`

Assuming I'm right, if you want to go through the effort, I used to control this like this: https://github.com/getsentry/sentry-dotnet/blob/f8a05a438452967e1408094dd04a6fbea6105f46/test/Sentry.Tests/Sentry.Tests.csproj#L29-L40

Reply:
Aha... that reminds me [about this](https://github.com/getsentry/sentry-dotnet/blob/e49c3c78202b5e43e4bca662f75a92b0f1b11360/test/Sentry.OpenTelemetry.Tests/SentrySpanProcessorTests.cs#L138-L144).

There are various tests for pretty critical functionality that we don't currently run on anything under .NET 5.0 then... and we can't guarantee our integration will work on anything less (in fact, I suspect it won't... due to an underlying issue in the OpenTelemetry .NET SDK).

Reply:
I've added a separate chore/issue for this: https://github.com/getsentry/sentry-dotnet/issues/2459